app-typescript/components/TreeSelect/TreeSelect.tsx

```diff
@@ -273,6 +274,11 @@ export class TreeSelect<T> extends React.Component<IProps<T>, IState<T>> {
                 }
             });
 
+            this.dropdownRef.current?.addEventListener('mousedown', (e: MouseEvent) => {
```

Comment:
Is this one to prevent date picker closing after selecting a month/year? If so the code should go to date picker - not tree select.